settings.gradle.kts

```diff
@@ -23,11 +23,18 @@ pluginManagement {
     }
 }
 
+buildscript {
+    repositories { mavenLocal() }
+    dependencies {
+        classpath("org.gradle.enterprise:gradle-enterprise-conventions-plugin:0.1-SNAPSHOT")
+    }
+}
+
 plugins {
     id("com.gradle.enterprise").version("3.3.4")
 }
 
-apply(from = "gradle/build-cache-configuration.settings.gradle.kts")
+apply(plugin = "org.gradle.enterprise.gradle-enterprise-conventions-plugin")
```

Comment:
:x: We should use `plugins {}` block (and `pluginManagement {}`) instead of `buildscript` and `apply(plugin = ...)`.

:x: We should not use `mavenLocal()`. It is a global state on the machine used for different things, which can lead to a ton of problems. We really only support `mavenLocal()` in Gradle for interoperability with Maven.

Instead, define a local repository that we use. Like `maven { name = "pluginsLocal"; url = "local-plugins-repo") }` and add the repo folder to .gitignore